util.py

```diff
@@ -87,6 +87,17 @@ def existingsnippet_monday(today):
     return end_monday.date()
 
 
+def _backfill_missing_snippets(user, all_snippets,
+                               current_monday, first_allowed_monday):
+    monday_ptr = current_monday - datetime.timedelta(7)
+    while monday_ptr - first_allowed_monday >= datetime.timedelta(0):
```

Comment:
Can you do `while monday_ptr >= first_allowed_monday`?

Reply:
Done.